### DISCUSSION TITLE
MacOS

### DISCUSSION THREAD

**Discussion by anubop** [NVIDIA-Omniverse/PhysX#13] [Q&A] [answered]
Hi! Am I able to run any code snippets with my M1 Macbook?

**Comment by amoravanszky** [comment, marked as answer]
I think you would need to do some porting work to get it to build on macOS.  I don't think such a port would be particularly difficult for you to do.  We do support ARM processors on Linux.  We would welcome a community contribution that makes it work on Macbooks.

**Comment by moraaar** [comment]
Hi, I've added a PR adding macOS support (both intel and arm architectures).
I hope the PR can be accepted. Any questions do not hesitate to ask.

Here is the PR: https://github.com/NVIDIA-Omniverse/PhysX/pull/51

Cheers,
Aaron
